frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from "../types/LogRecordTypes";
 
 const countLogRecords = async ({
-  building = "",
+  buildingId,
```

Comment:
should there be a default here in case we don't want to pass in/filter by a building ID?